### DISCUSSION TITLE
Login with Azure-ad-b2c url

### DISCUSSION THREAD

**Discussion by erhankaradeniz** [nextauthjs/next-auth#1451] [Help]
**Your question**
Is it possible to also login with a business account?

**What are you trying to do**
I'm trying to login with a business account. Personal account works fine however because next-auth uses this format for the authorization url `authorizationUrl: "https://login.microsoftonline.com/".concat(tenant, "/oauth2/v2.0/authorize?response_type=code&response_mode=query")` it is not possible to login unless you are within that tenant. if the url becomes: `https://login.microsoftonline.com/common/oauth2/v2.0/authorize?response_type=code&response_mode=query` which microsoft suggests for allowing all logins. it works fine.

Is there any reason for choosing that url structure, is it to limit the costs?

**Comment by balazsorban44** [comment]
So as I  an see, if you don't set `tenantId` it will default to `common` https://github.com/nextauthjs/next-auth/blob/582520f8ef5bd029143456c9161147ae644aef72/src/providers/azure-ad-b2c.js#L2

By the way, you can override as many of the provider options as you want, you could simply add your own authorizationUrl under clientId and clientSecret
